lib/matterStacks.ts

```diff
@@ -633,9 +644,8 @@ export class MatterStack extends Stack {
     // Creates the S3 bucket that will hold the CRLs for the Matter PKI.
     private createMatterCrlBucket(bucketId: string, crlBucketName: string): Bucket {
         const matterCrlBucket = new Bucket(this, bucketId, {
-            autoDeleteObjects: true,
             versioned: false,
-            bucketName: crlBucketName,
+            //bucketName: crlBucketName,
```

Comment:
should we just remove this rather than commenting it out?

Reply:
oh, commented to test something and forgot to revert. Thanks!